feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Timetable.kt

```diff
@@ -359,6 +372,11 @@ class ScreenScrollState(
     val maxY: Float
         get() = _scrollY.lowerBound ?: 0f
 
+    val isScrollYProgress
+        get() = _isScrollYProgress.value || _scrollY.isRunning
+    val isScrollXProgress
+        get() = _isScrollXProgress.value || _scrollX.isRunning
+
```

Comment:
Get the state during a drag or scroll animation.

- `_scrollY(Animatable).isRunning` returns false while dragging the screen
- Set `_isScrollYProgress` to true while dragging.